Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -46,6 +46,7 @@ def cmd_output(*args)
       ENV["HOMEBREW_BREW_FILE"] = HOMEBREW_PREFIX/"bin/brew"
       ENV["HOMEBREW_INTEGRATION_TEST"] = cmd_id_from_args(args)
       ENV["HOMEBREW_TEST_TMPDIR"] = TEST_TMPDIR
+      ENV["HOMEBREW_NO_GITHUB_API"] = "1"
```

Comment:
Can we move this to `cmd/tests.rb` and hide it behind an `--offline` argument? I'm not too happy disabling this unconditionally and only for the integration tests (some other parts of the tests might exercise the GitHub API in the future) as I consider this to be a temporary measure until we have a better solution for offline testing.